src/index.js

```diff
@@ -0,0 +1,24 @@
+import Weather from './weather';
+import Dom from './views';
+
+const weather = new Weather('Harare');
+const dom = new Dom();
+
+function getWeather() {
+  weather.getWeather().then((results) => {
+    dom.show(results);
+  });
+}
```

Comment:
According to the previous review, @Torres-ssf mentioned that the `toggle button` is not working properly, kindly look into this, still not working.